go.mod

```diff
@@ -1,12 +1,12 @@
-module github.com/calyptia/enterprise-plugin-dummy
+module github.com/chronosphereio/calyptia-core-fluent-bit-dummy
 
-go 1.17
+go 1.21.0
 
-require github.com/calyptia/plugin v0.1.7
+toolchain go1.21.6
```

Comment:
does 1.21.6 have the fix for the calling convention used by the plugin layer? that would be great. If so we should enforce that version.

Reply:
Go 1.21.7 could be included the fix: https://go-review.googlesource.com/q/project:go+branch:release-branch.go1.21